ui/src/alerting/actions/notificationRules.ts

```diff
@@ -0,0 +1,88 @@
+// Libraries
+import {client} from 'src/utils/api'
+import {Dispatch} from 'react'
+
+// Constants
+import * as copy from 'src/shared/copy/notifications'
+
+//Actions
+import {
+  notify,
+  Action as NotificationAction,
+} from 'src/shared/actions/notifications'
+
+// Types
+import {RemoteDataState} from '@influxdata/clockface'
+import {NotificationRule, GetState} from 'src/types'
+
+export type Action =
+  | ReturnType<typeof setAllNotificationRules>
+  | ReturnType<typeof setNotificationRulesStatus>
+  | ReturnType<typeof setNotificationRule>
+  | ReturnType<typeof setNotificationRuleStatus>
+
+const setAllNotificationRules = (
+  status: RemoteDataState,
+  notificationRules?: NotificationRule[]
+) => ({
+  type: 'SET_ALL_NOTIFICATIONRULES' as 'SET_ALL_NOTIFICATIONRULES',
```

Comment:
ahhhh please can we have one more underscore :)

Reply:
I want to just get this in now, will change in next PR promise :)